app/utils/CheckYourAnswersHelper.scala

```diff
@@ -30,6 +30,16 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
 
   def getTrusteeFirstLastName(index: Int): String = userAnswers.get(TrusteesNamePage(index)).get.toString
 
+  def trusteeLiveInTheUK(index : Int): Option[AnswerRow] = userAnswers.get(TrusteeLiveInTheUKPage(index)) map {
+    x =>
+      AnswerRow(
+        "trusteeLiveInTheUK.checkYourAnswersLabel",
+        yesOrNo(x),
+        routes.TrusteeLiveInTheUKController.onPageLoad(CheckMode, index).url,
+        getTrusteeFirstLastName(index)
```

Comment:
I refactored this in TRUS-337 to be in the companion object. Has this changed back?

Reply:
You also haven't used this method anywhere. Can you update the TrusteesAnswerController to implement this.

Reply:
Fixed